website/docs/docs/build/python-models.md

```diff
@@ -443,9 +515,10 @@ def model(dbt, session):
 
 #### Code reuse
 
-Currently, Python functions defined in one dbt model cannot be imported and reused in other models. This is something wdbt would like to support, so there are two patterns we're considering:
-1. Creating and registering **"named" UDFs** &mdash; This process is different across data platforms and has some performance limitations. (Snowpark does support ["vectorized" UDFs](https://docs.snowflake.com/en/developer-guide/udf/python/udf-python-batch.html): Pandas-like functions that can be executed in parallel.)
-2. **Private Python packages** &mdash; In addition to importing reusable functions from public PyPI packages, many data platforms support uploading custom Python assets and registering them as packages. The upload process looks different across platforms, but your code’s actual `import` looks the same.
+Currently, you can't import or reuse Python functions defined in one dbt model in other models. This is something dbt Labs would like to support, so there are two patterns we're considering:
```

Comment:
> defined in one dbt model in other models
The new syntax makes this really hard for me to parse. I actual prefer the original way this sentence was worded. The rest of the change is lovely.